src/components/Comments/ReplyCommentItem.js

```diff
@@ -177,7 +193,7 @@ const ReplyCommentItem = ({
         const members = [comment?.user?.id, profile.myProfile.user_id];
         await createSignChat(members, selectedUser);
       } else {
-        await sendMessageDM(comment?.id, 'comment', 'SIGNED');
+        await sendMessageDM(comment?.id, 'comment', 'SIGNED', feedId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendMessageDM` function call has been updated to include the `feedId`. This change is necessary for the new feature being implemented.
```diff
+ await sendMessageDM(comment?.id, 'comment', 'SIGNED', feedId);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->